tests/test_route53/test_route53.py

```diff
@@ -390,6 +403,21 @@ def test_hosted_zone_private_zone_preserved_boto3():
     len(hosted_zones["HostedZones"]).should.equal(1)
     hosted_zones["HostedZones"][0]["Config"]["PrivateZone"].should.equal(True)
 
+    # create_hosted_zone statements with  PrivateZone=True,
+    # but without a _valid_ vpc-id should fail.
+    conn = boto3.client("route53", region_name=region)
+    with pytest.raises(KeyError) as e:
```

Comment:
Moto in ServerMode will only throw exceptions of type `ClientError`, so we should catch that instead.
The assertion then becomes something like `e.value.response["Error"]["Message"] == expected_msg`

Changing this should get the tests to pass

Reply:
Oh, okay.  I'm not sure how to test that locally on my laptop. I'll give it a whirl. Do I just fire up the moto_server and then just run the server tests ?

Reply:
Firing up moto_server to start it, yes. Then you can use `TEST_SERVER_MODE=true pytest tests/test_route53` to run the same tests against that server.

Reply:
Awesome, that's what I suspected. Looks like the Makefile has a `test_server` target as well, so I'm assuming I just need to fire it up on port 5000 (assuming the developer FAQ is current).

I'll see what I can do tonight.